src/Microsoft.XmlSerializer.Generator/tests/CodeAnalysis.ruleset

```diff
@@ -0,0 +1,239 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why do we suppress code analysis for tests all together? If it's for the baseline file, can we only exclude that file?